src/lm_buddy/paths.py

```diff
@@ -68,16 +68,16 @@ def validate_asset_path(path: str) -> "AssetPath":
 
 def format_file_path(path: str | Path) -> AssetPath:
     path = Path(path).absolute()
-    return f"{PathPrefix.FILE}{path}"
+    return f"{PathPrefix.FILE.value}{path}"
```

Comment:
We need to have a unit test to validate these kind of functions. Do we have any?

Reply:
Yes! I think they were essential to catch these issues, and I'll make sure to create more going on